### ISSUE TITLE
wonder about valid_loss 

### ISSUE THREAD

**Issue by sjk0825** [XMUDM/BASTS#2]
hi, i do experiment basts and have some question about valid loss.

in experiment, valid_loss decrease for 100 epochs. and i saw below code.

>>
loss = run_epoch_([val_batches[0]], model,
                              MultiGPULossCompute(model.generator, criterion, devices=devices, opt=model_opt))
>>

i think **opt=model_opt** make model train at valid dataset either.

so when i experiments with opt=None , valid loss increased in some epoch ( between 10~20 epoch ).

my question is that.. 
What method does the performance of paper follow?
